packages/Swiper/index.tsx

```diff
@@ -25,7 +68,6 @@ export const Swiper = forwardRef((props, ref) => {
     slidesToSwipe,
     ...rest
   } = props
-
   const translateX = -(pageIdx * 100)
 
   useEffect(() => {
```

Comment:
On the lines below, should we just remove support for immutable objects since it makes no sense in the library ? And then let immutable users to use .toJS util in order to have a real javascript array ?

Reply:
the link to the line (could not comment on unchanged parts 😅 ): https://github.com/WTTJ/welcome-ui/pull/883/files#diff-93657189125bc2e8fe169d4d98cbf890e29e16e1df9c9d2427fd455314cec5cdR75